src/main/webapp/resources/js/admin-deactivate-room-button.js

```diff
@@ -0,0 +1,22 @@
+$('.for-table tbody').on('click', '#deactivateButton', function() {
```

Comment:
This must not work. Because you established many element with the same id = deactivateButton. Only first button may work.